OpenRA.Mods.Common/Widgets/UnitCommandWidget.cs

```diff
@@ -129,36 +130,26 @@ bool PerformDeploy()
 			return true;
 		}
 
-		bool PerformStanceCycle()
+		bool SetUnitStance(UnitStance unitStance)
 		{
 			var actor = world.Selection.Actors
```

Comment:
The `PerformKeyboardOrderOnSelection` doesn't depend on this, so you can remove it and the check below.